pontoon/api/tests/test_views.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+
+from rest_framework.test import APIClient
+
+from pontoon.base.models.project import Project
+from pontoon.base.models.translation_memory import TranslationMemoryEntry
+from pontoon.terminology.models import Term, TermTranslation
+from pontoon.test.factories import LocaleFactory, ProjectFactory
+
+
+@pytest.fixture
```

Comment:
As discussed at the call, let's drop the fixtures and move the code to the tests where they are actually used, even if that means the code will be repeated.